src/export.ts

```diff
@@ -30,6 +25,42 @@ export async function generateVCard(c: Contact) {
   // https://datatracker.ietf.org/doc/html/rfc6350#section-3.3
   vcf += `X-CALLSIGN:${c.callsign}\n`;
 
+  vcf += `X-STATION-TYPE:${c.cardType}\n`;
+
+  // this is hopefully not that wrong
+  if (c.cardType == "repeater") {
+    
+    // using `X-LOCATION` instead of `ADR` or `GEO` since the inputted location could be something like "UI Steam Plant, Moscow, ID" - which isn't a mailing address, and GEO wants lat/long
```

Comment:
IMO it's better to use `ADR` since it'll actually have interoperability, even if it's not _quite_ a street address. if you invent something new like `X-LOCATION`, no other .vcf parser (contacts app) will be able to read it